include/deal.II/matrix_free/tensor_product_point_kernels.h

```diff
@@ -0,0 +1,1601 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2017 - 2024 by the deal.II authors
```

Comment:
Can you please write the appropriate date when this part of the code was first written,
```suggestion
// Copyright (C) 2020 - 2024 by the deal.II authors
```
(not sure if it is retained by the automatic scripts).